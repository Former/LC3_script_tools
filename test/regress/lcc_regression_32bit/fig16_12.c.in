1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word
