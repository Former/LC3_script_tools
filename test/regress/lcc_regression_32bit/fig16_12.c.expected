Input a word (less than 20 characters): 1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word1234567890Input a word
The word contains 968 characters
